sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 # Release History
 
-## 1.4.0b2 (Unreleased)
+## 1.5.0 (Unreleased)
```

Comment:
what's the plan for the release? If we are going to ship it before the bot GA then let's make sure the bot identififier deprecation is mentioned here, also we have to get approval for deprecation API changes

Reply:
Yes, I believe we would be releasing this before bot GA. So you mean, updating the changelog to mention deprecation of bot identifier changes and getting approval on this PR from Python architects ?

Reply:
It'll get updated once you will merge to the main branch. Just make sure getting approval from architects for the apiview generated from the main branch which will include also deprecation changes